service/src/main/kotlin/io/provenance/explorer/domain/entities/Governance.kt

```diff
@@ -41,7 +41,7 @@ object GovProposalTable : IntIdTable(name = "gov_proposal") {
 
 fun String.getProposalType() = this.split(".").last().replace("Proposal", "")
 
-fun Message.toObjectNode(protoPrinter: JsonFormat.Printer) =
+fun Message.toPropsalTitleAndDescrip(protoPrinter: JsonFormat.Printer) =
```

Comment:
Typeo in `toPropsalTitleAndDescrip`. 

Would it make more sense to use full name for all the words in the method name? `toProposalTitleAndDescription`

Reply:
Updated